mmv1/products/clouddeploy/terraform.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  DeliveryPipeline: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    id_format: '{{name}}'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "deploy_delivery_pipeline"
+        primary_resource_id: "pipeline"
+        vars:
+          name: "dev"
+    properties:
+      name: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+      region: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+  Target: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    id_format: '{{name}}'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "deploy_target"
+        primary_resource_id: "pipeline"
+        vars:
+          name: "tf-test"
+    properties:
+      name: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+      region: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Why do we need default from api on region?

Reply:
It is computed field.